skops/card/default_template.md

```diff
@@ -29,6 +29,12 @@ The model plot is below.
 
 {{ model_plot }}
 
+## Evaluation Results
+
+Below are the evaluation results.
```

Comment:
I think the user should also set the text of this section, explaining how the metrics are calculated. For example, are they from a cross validate call, a held out validation set, or a certain part of the dataset.